casanovo/casanovo.py

```diff
@@ -146,91 +145,151 @@ def sequence(
+            "caspase 8",
+            "caspase 9",
+            "caspase 10",
+            "chymotrypsin high specificity",
+            "chymotrypsin low specificity",
+            "clostripain",
+            "cnbr",
+            "enterokinase",
+            "factor xa",
+            "formic acid",
+            "glutamyl endopeptidase",
+            "granzyme b",
+            "hydroxylamine",
+            "iodosobenzoic acid",
+            "lysc",
+            "ntcb",
+            "pepsin ph1.3",
+            "pepsin ph2.0",
+            "proline endopeptidase",
+            "proteinase k",
+            "staphylococcal peptidase i",
+            "thermolysin",
+            "thrombin",
+            "trypsin",
+            "trypsin_exception",
+        ]
+    ),
+    default="trypsin",
 )
 @click.option(
-    "-v",
-    "--verbosity",
-    help="""
-    Set the verbosity of console logging messages. Log files are
-    always set to 'debug'.
-    """,
+    "--digestion",
+    help="Digestion: full, partial",
     type=click.Choice(
-        ["debug", "info", "warning", "error"],
+        ["full", "partial"],
         case_sensitive=False,
     ),
-    default="info",
+    default="full",
 )
-def annotate(
-    peak_path: str,
-    tide_path: str,
-    output: Optional[str],
-    verbosity: str,
-) -> None:
-    """Annotate a given .mgf with candidates as selected by a Tide search for Casanovo-DB.
-
-    PEAK_PATH must be one MGF file from which to annotate spectra.
-
-    TIDE_PATH must be one directory containing the Tide search results of the <PEAK_PATH> .mgf.
-    This directory must contain tide-search.decoy.txt and tide-search.target.txt
-    """
-    if output is None:
-        output = setup_logging(output, verbosity)
-        logger.info(
-            "Output file not specified. \
-            Annotated MGF will be saved in the same directory \
-            as the input MGF."
-        )
-        output = peak_path.replace(".mgf", "_annotated.mgf")
-    else:
-        output = setup_logging(output, verbosity)
-
-    annotate_mgf(peak_path, tide_path, output)
-
-    logger.info("DONE!")
-
-
-@main.command(cls=_SharedParams)
-@click.argument(
-    "peak_path",
-    required=True,
-    nargs=-1,
-    type=click.Path(exists=True, dir_okay=False),
+@click.option(
+    "--missed_cleavages",
+    help="Number of allowed missed cleavages",
+    type=int,
+    default=0,
+)
+@click.option(
+    "--max_mods",
+    help="Maximum number of modifications per peptide",
+    type=int,
+    default=0,
+)
+@click.option(
+    "--min_length",
+    help="Minimum peptide length",
+    type=int,
+    default=6,
+)
+@click.option(
+    "--max_length",
+    help="Maximum peptide length",
+    type=int,
+    default=50,
+)
+@click.option(
+    "--precursor_tolerance",
+    help="Precursor tolerance window size (ppm)",
+    type=int,
+    default=20,
+)
+@click.option(
+    "--isotope_error",
+    help="Isotope error levels to consider (list of ints, e.g: 1,2)",
```

Comment:
We could use the information from the config file to populate this, but I find it confusing to have some important Casanovo-DB configuration options within the CLI and decoupled from Casanovo and have some be hidden in the config file and affect both Casanovo and Casanovo-DB. I opted to have most db-search related options stuck solely to the command line to prevent confusion. Let me know if things should be migrated one way or another.

I also updated the help message.